circle.yml

```diff
@@ -8,17 +8,12 @@ dependencies:
   cache_directories:
     - ~/.cache/yarn
 
-general:
-  branches:
-    only:
-      - release
-
 test:
   override:
     - yarn build
 
 deployment:
   release:
-    branch: release
+    branch: master
```

Comment:
I am pretty sure this will work already given `circle.sh` already commits and pushes to both `master` and `release`

Lets test it @GHaberis 😄